Fixed #35748 -- Documented that Field.formfield() may return None. (#18570)
django/django: docs/howto/custom-model-fields.txt

a custom form field (and even a form widget). See the :doc:`forms documentation
</topics/forms/index>` for information about this.

> If you wish to ignore the field in the :class:`~django.forms.ModelForm`, you
> can return ``None`` from the :meth:`~Field.formfield` method.

Reviewer: ```suggestion
If you wish to exclude the field from the :class:`~django.forms.ModelForm`, you
can override the :meth:`~Field.formfield` method to return ``None``.
```
I think exclude might be clearer than ignore

Continuing our ongoing example, we can write the :meth:`~Field.formfield` method

> as follows::

Reviewer: ```suggestion
as::
```
I don't think we need to add the follows

    class HandField(models.Field):
        # ...

        def formfield(self, **kwargs):

>             # To ignore the field in the ModelForm if some condition is met:

Reviewer: ```suggestion
            # To exclude the field in the ModelForm if some condition is met:
```

>             # To exclude the field from the ModelForm if some condition is met:
>             some_condition = kwargs.get("some_condition", False)
>             if some_condition:
>                 return None
> 
>             # To set up some defaults while letting the caller override them:

Reviewer: ```suggestion
            # Exclude the field from the ModelForm when some condition is met.
            some_condition = kwargs.get("some_condition", False)
            if some_condition:
                return None

            # Set up some defaults while letting the caller override them.
```
(nit)
